catalog/citation/admin.py

```diff
@@ -23,21 +21,14 @@ def lookups(self, request, model_admin):
         human-readable name for the option that will appear
         in the right sidebar.
         """
-        
-
-        return (
-            tuple([(s[0], s[0]) for s in Publication.Status])
-
-        )
+        return (tuple([(s[0], s[0]) for s in Publication.Status]))
```

Comment:
What's the difference between this and `return tuple([(s[0], s[0]) for s in Publication.Status])`?